### DISCUSSION TITLE
<Copilot and Pycharm>

### DISCUSSION THREAD

**Discussion by fmusella** [community/community#46070] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hi everyone!

Similar issues have been posted by other users, but I couldn't find a solution that worked for me so I am posting it again.

I am trying to install GitHub Copilot on Pycharm Pro. I am following this guide: https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-a-jetbrains-ide.

When I reach the final step and I have to authenticate my GitHub account I get the following error:
> The login to GitHub failed. Reason: Error: EACCES: permission denied, mkdir '/Users/***/.config/github-copilot', request id: 6, error code: 1001.

I have tried the following:

- Uninstall and reinstall the plugin;
- Install the plugin from disk;
- Verify that I am all set for Copilot in my GitHub account.

I don't know what to try anymore. Anyone has a solution?

Versions: PyCharm: 2022.3.2, GitHub Copilot: 1.2.1.2347, OS: macOS Monterey 12.4.

Thank you!

**Comment by vaskers5** [comment]
the same trouble

**Comment by vaskers5** [comment]
Fixed like this:
makedir /Users/***/.config/github-copilot
touch /Users/***/.config/github-copilot/hosts.json 
sudo chmod go+rwx /Users/***/.config/github-copilot/hosts.json

**Reply by mrfear-41** [reply]
how to do this on windows?
